### PR TITLE
#992 Add documentation for mbeans configuration

### PR DESCRIPTION
**Pull request by ljacomet**

### CODE REVIEW

File: docs/src/docs/asciidoc/user/107.adoc [hunk 1]
```diff
@@ -84,6 +84,22 @@ NOTE: You can also use the `CachingProvider.getCacheManager()` method that takes
...
+When using Ehcache XML, you may want to enable management and / or statistics MBeans for JSR-107 caches.
+
+You can do this at two different levels:
+
+[source,xml,indent=0]
+----
+include::{sourcedir}/107/src/test/resources/org/ehcache/docs/ehcache-107-mbeans-cache-manager-config.xml[lines=17..-1]
+----
+
+<1> Using the JSR-107 service extension, you can enable MBeans by default
```
Comment (lorban) [inline docs/src/docs/asciidoc/user/107.adoc]: MBeans *and* statistics.

**Comment by lorban** [comment]
LGTM

**Comment by AbfrmBlr** [comment]
If i am reading the network right it needs a rebase against 3.0.

**Comment by ljacomet** [comment]
Did a minor addition to the doc.

As for the rebase, no as it needs to be merged to master as well in a separate PR.
I am long overdue for a mail to ehcache-dev indicating the bug fixing strategy we decided upon.

**Pull request merged**